### ISSUE TITLE
Limbo/Discord/Lack of information

### ISSUE THREAD

**Issue by mattba10** [Corail31/tombstone_lite#135]
### Minecraft Version

1.19.3

### Forge Version

43.2.3

### Corail Tombstone Version

8.2.4-1.19.2

### What happened?

Trying to complete all achievements in the pack (ATM8). I have found it a struggle to complete corail and I am actually finding it frustrating the lack of information out there. I understand you don't want to just give it away but it means that im less likely to recommend or play the mod - im sure im not alone in this.

Could I please seek advice on both the Limbo and Survive darkness achievements - I havn't found information out there and was hoping there may be a discord?

The darkness I have heard is akin to the Gru - which isn't in the pack ATM8

### Gametype

Dedicated Server

### Happen by testing with only Corail Tombstone (required)

Yes

### Occurence

occurs all the time

### Tested without any Forge "alternatives" (Magma, Mohist, and so on)

- [X] Yes

### Valid Launcher

- [X] Yes

**Comment by Corail31** [comment]
Hi, please keep your questions for the CurseForge page, this github is only for issues as explained on the form you followed.
The advancement 'Limbo' is only if you lose knowledge of death points during a death (it can happen if you die from the damages of a 'Familiar Receptacle' when it recovers, more precisely if you die from the BeyondTheGrave damage).
The other one has been explained multiple times on the CurseForge page (protect against wither damage, against BeyondtheGrave damages and others dark things).
I also don't think advancements are done to finish them all (it depends more on your alignment for some of them, or the items that you use) and i don't have time for a 'discord server' as i often mention, the compendium provides already lots of infos (but not all obviously).

**Comment by Twisted-Code** [comment]
While I understand your intent in focusing on actual bugs, you seem to be resistant to the realization that if people are still asking questions, your existing documentation may not be sufficient. For instance:
1. how do I actually get a familiar? I see that there is a perk about "Pray of familiar" when I have about one segment of positive alignment, but nothing in the compendium actually explains... what I need to do. I guess I can assume that since it's a prayer, I use the ankh, but what do I do? Do I just go point it at a random passive mob and hope that's all I need? Do I need any of the familiar-related items like receptacle or impregnated diamond? The guidance is kind of lacking, so I'm left to fall back on [#tryitandsee](https://tryitands.ee/) and hope my assumptions are right. (to be fair, the assumption was in fact right. But I still feel like it was a bit of a blind leap, more than I normally like to make)
2. once I have a familiar, what do I actually do with it? I can't find anything about familiars giving any sort of special ability like the familiars in Ars Nouveau. The only thing that seems clearly possible is reviving them using either a soul receptacle (which is also not explained clearly, though I think that's at least partly a matter of English fluency) or one of those blue shards.
3. what exactly do I do with blue souls? I "discovered" that apparently I'm not supposed to use them for things where normal souls would suffice, but I already kind of figured that when I noticed it was a different color and couldn't figure out what to do with it, so I just used it like a normal soul.
